File: selection.py
def selection_sort(arr):
    for i in range(len(arr)):
        min_index = i
        # Finding the min element
        for j in range(i + 1, len(arr)):
            if arr[j] < arr[min_index]:
                min_index = j

                #Swapping 
        arr[i], arr[min_index] = arr[min_index], arr[i]
    return arr 

print(selection_sort([10, 7, 8, 9, 1, 5]))

File: selection.c
#include <stdio.h>

void selection_sort(int arr[], int n)
{
    for (int i = 0; i < n; i++)
    {
        int min_index = i;
        for (int j = i + 1; j < n; j++)
        {
            if (arr[j] < arr[min_index])
            {
                min_index = j;
            }
        }
        int temp = arr[i];
        arr[i] = arr[min_index];
        arr[min_index] = temp;
    }
}

int main()
{
    int arr[] = {10, 7, 8, 9, 1, 5};
    int n = sizeof(arr) / sizeof(arr[0]);

    selection_sort(arr, n);

    printf("Sorted array: ");
    for (int i = 0; i < n; i++)
    {
        printf("%d ", arr[i]);
    }
    printf("\n");

    return 0;
}
